lib/src/web_socket_handler.dart

```diff
@@ -65,8 +64,8 @@ class WebSocketHandler {
     }
 
     var protocol = _chooseProtocol(request);
-    request.hijack((untypedChannel) {
-      var channel = (untypedChannel as StreamChannel).cast<List<int>>();
+    request.hijack((typedChannel) {
+      var channel = typedChannel.cast<List<int>>();
```

Comment:
Now that I think about it, this `cast()` call shouldn't be necessary at all, since in 0.7 [`hijack()`](https://github.com/dart-lang/shelf/blob/4c5ea1c108d48b964a5027bacef60724118d73ce/lib/src/request.dart#L244) is typed to pass a `StreamChannel<List<int>>`. So you can just have this callback take `channel` and avoid the intermediate assignment altogether.